src/lib/operators/operator_expression.cpp

```diff
@@ -0,0 +1,77 @@
+#include "operator_expression.hpp"
+
+#include "constant_mappings.hpp"
+#include "logical_query_plan/abstract_lqp_node.hpp"
+#include "logical_query_plan/lqp_expression.hpp"
+#include "utils/assert.hpp"
+
+namespace opossum {
+
+std::shared_ptr<OperatorExpression> OperatorExpression::create_column(const ColumnID column_id,
+                                                                      const std::optional<std::string>& alias) {
+  auto expression = std::make_shared<OperatorExpression>(ExpressionType::Column);
+  expression->_column_id = column_id;
+  expression->_alias = alias;
+
+  return expression;
+}
+
+OperatorExpression::OperatorExpression(const std::shared_ptr<LQPExpression>& lqp_expression,
```

Comment:
This looks like part of translating an LQP to a PQP. As such, it is surprising to find it in `src/lib/operators`.

Reply:
tiny little uncleanness, but better than having a PQPExpression constructor that takes all the class' members as args and then put a translation methods that passes them in into the LQPTranslator - imo.